### DISCUSSION TITLE
Portable apps

### DISCUSSION THREAD

**Discussion by KnifMelti** [Romanitho/Winget-Install-GUI#25] [Ideas]
Instead of on the Admin page let you install two of the Nirsoft Utils you could point to [WSCC](https://www.kls-soft.com/wscc/) ([Portable x64](https://www.kls-soft.com/downloads/wscc_x64.zip))
Then we could install (or run over http directly) **anything** from a lot of sources...

**Comment by KnifMelti** [comment]
https://github.com/Romanitho/Winget-Install-GUI/pull/26

**Reply by KnifMelti** [reply]
...and maybe let the users themselves decide if they need the stuff on the tab **Admin Tools** or not (no more hiding requiring F10 to show)?

**Reply by Romanitho** [reply]
In that case we don't need to do more 😅
We just remove the admin part and if someone needs admin tools, get them aside.

**Reply by KnifMelti** [reply]
No, **WiGui** is an **excellent** place to gather them all in!
Just show the Admin Tools tab by default so that everyone can take advantage of them too.

**Comment by Romanitho** [comment]
WiGui is on Winget, just noticed ^^ as a portable app of course. Doc updated
